template/studio/schemas/documents/route.js

```diff
@@ -87,12 +78,33 @@ export default {
       name: 'disallowRobots',
       type: 'boolean',
       fieldset: 'visibility'
-    }
-    */
+    },
+    {
+      name: 'queries',
+      type: 'array',
+      description: 'Search queries to match',
+      of: [
+        {
+          type: 'string'
+        }
+      ],
+      options: {
+        layout: 'tags'
+      }
+    },
+    {
+      name: 'campaign',
+      type: 'string',
+      title: 'Campaign',
+      description: 'UTM for campaings'
+    },
+
+    {
+      name: 'experiment',
+      type: 'experiment',
+      description: 'Use this to A/B/n test this route towards different pages',
+    },
```

Comment:
There is nothing handling this stuff in the frontend, so I think its confusing to include it

Reply:
Same with opengraph, sitemap etc

Reply:
I think it’s fine – we’ll just caveat it in the readme. The point here is to show off the studio features.

Reply:
It has been a problem in the past. They see "search queries" and get excited and start asking about our support for end user text search. Which I then have to explain we do not really have much support for. Thats why its not included.

Reply:
Ok. I thought this was for personalization purposes, that is, when you want to add functionality where you deliver certain content based on the adword search term https://www.evergage.com/blog/personalize-website-based-adwords-search-terms/